xwiki-commons-core/xwiki-commons-xml/src/test/java/org/xwiki/xml/internal/html/DefaultHTMLCleanerTest.java

```diff
@@ -125,7 +125,9 @@ void specialCharacters()
         assertHTML("<p>\"&amp;</p>", "<p>\"&</p>");
         assertHTML("<p><img src=\"http://host.com/a.gif?a=foo&amp;b=bar\" /></p>",
             "<img src=\"http://host.com/a.gif?a=foo&amp;b=bar\" />");
-        assertHTML("<p>&#xA;</p>", "<p>&#xA;</p>");
+        assertHTML("<p>\n</p>", "<p>&#xA;</p>");
+        // the following is not a "special" character, but any ole unicode character
```

Comment:
Is this commend missing something ? Did you mean "any ole unicode character is escaped" ?

Reply:
The comment was part of changes performed by @ClemensRobbenhaar in the original PR. Honestly I don't know what's an "ole unicode character" so not sure how to fix the comment...

Reply:
I think "ole unicode character" is related to characters with the two dot on top of it.